create_per_gene_matrix.py

```diff
@@ -1,17 +1,50 @@
 from gnomad_qc.v2.resources import get_gnomad_meta, get_gnomad_data, annotations_ht_path
+from gnomad.resources.grch37.gnomad import public_release
 import hail as hl
 from typing import List, Union
 import argparse
 from resources import *
 
-CHET_THRESHOLD = 0.505
-SAME_HAP_THRESHOLD = 0.0164
+CHET_THRESHOLD = 0.55
+SAME_HAP_THRESHOLD = 0.1
+ALLELE_FREQUENCY_CUTOFFS = [0.01] # 0.05, 0.02, 0.015, 0.01, 0.005, 0.001, 0.0005, 0.0001, 0.00005, 0.00001
```

Comment:
Maybe the list of allele frequency cutoffs should be an argument that gets passed to the script. Then when writing out the files add AF into the name somehow. Then run some AF cutoffs in parallel and provide another argument in the script for merging AF tables into one? Just trying to think of the best way to get around the error when there are too many allele frequencies.